lib/mix/lib/mix/release.ex

```diff
@@ -802,4 +809,16 @@ defmodule Mix.Release do
         error
     end
   end
+
+  @spec parse_strip_beams_option(true | false | Keyword.t()) :: {true | false, List.t() | nil}
+  def parse_strip_beams_option(option) do
```

Comment:
I think this is a private function 🙂 

```suggestion
  defp parse_strip_beams_option(option) do
```